src/components/StepIndicator/StepIndicator.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import StepIndicator, {StepIndicatorProps} from '.';
+
+const baseProps: StepIndicatorProps = {
+  number: 1,
+  text: <div>Hello world</div>,
+};
+
+describe('StepIndicator', () => {
+  it('renders proper default className', () => {
+    render(<StepIndicator {...baseProps} />);
+    const el = screen.getByTestId('StepIndicator');
+
+    expect(el.className).toBe('StepIndicator');
+  });
+
+  it('renders with classNames passed in', () => {
+    render(<StepIndicator className="test" {...baseProps} />);
+    const el = screen.getByTestId('StepIndicator');
+
+    expect(el.className).toContain('test');
```

Comment:
let's add tests here to make sure all the children div has correct classNames too

Reply:
done